pom.xml

```diff
@@ -72,8 +67,6 @@
     <license.mailto>mailto:info AT sonarsource DOT com</license.mailto>
 
     <artifactsToPublish>${project.groupId}:${project.artifactId}:hpi${project.groupId}:${project.artifactId}:json:cyclonedx</artifactsToPublish>
-
-    <maven.compiler.release>8</maven.compiler.release>
```

Comment:
Jenkins parent pom now has Java 11 as baseline and doesn't allow configuring older